src/generate/__tests__/__snapshots__/index.js.snap

```diff
@@ -9,8 +9,7 @@ Description
 These people contributed to the project:
 <!-- ALL-CONTRIBUTORS-LIST:START -->
 <!-- prettier-ignore -->
-| Kent C. Dodds is awesome! | Divjot Singh is awesome! | Jeroen Engels is awesome! |
-| :---: | :---: | :---: |
+<style>#emoji-table, #emoji-table td { border: 1px solid #ccc; }</style><table id=\\"emoji-table\\" cellspacing=0 cellpadding=1><tr><td>Kent C. Dodds is awesome!</td><td>Divjot Singh is awesome!</td><td>Jeroen Engels is awesome!</td></tr></table>
```

Comment:
Any reason for having `<style>` tags visible in the readme file?

Reply:
I'm not sure why on the Markdown renderer on VSCode it doesn't show up (as expected) but here it does 😕.